openlibrary/templates/authors/infobox.html

```diff
@@ -14,6 +14,11 @@
             <td><span itemprop="$itemprop">$value</span></td>
         </tr>
 
+$def render_social_icon(url, icon_url, title):
+    <a href="$url" title="$_('View on %(site)s', site=title)">
+        <img class="profile-icon" src="$icon_url" alt="">
```

Comment:
```suggestion
$def render_social_icon(url, icon_url, label):
    <a href="$url" title="$_('View on %(site)s', site=label)">
        <img class="profile-icon" src="$icon_url" alt="$label">
```
The `alt` property should describe/be a stand‐in for the image, so the social’s `label` is probably a good fit. (Also, changing the `title` to `label` to be consistent with how it’s named in the dict, since my first iteration of this had the function signature take an addition `label` property…)

Reply:
In this case, because we have the `title` tag on the above anchor element, the image can be seen as purely presentational, and doesn't require an alt tag. E.g. screen readers will read the `title` tag, I believe. But this is a bit of an edge case, so not entirely sure.